templates/admin/src/i18n/messages/MessageResolver.ts

```diff
@@ -14,16 +16,19 @@ export default class MessageResolver implements PlumeMessageResolver {
   }
 
   private messageResolver = (messageKey: string, ...messageArgs: string[]): string => {
-    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-    const translation = (this.messages.t() as any)[messageKey];
+    const translation = messageKey.split('.')
```

Comment:
Il faudrait commenter ce code, sinon on perd trop de temps en lecture

Reply:
`Enables to find the message value in a object (eg {a: {b: {c: value}}} with a string key (eg a.b.c)`

Reply:
je l'avais déjà fait au dessus de la fonction